src/goto-programs/xml_goto_trace.cpp

```diff
@@ -27,15 +27,15 @@ Author: Daniel Kroening
 
 xmlt full_lhs_value(const goto_trace_stept &step, const namespacet &ns)
 {
-  xmlt full_lhs_value{"full_lhs_value"};
+  xmlt value_xml{"full_lhs_value"};
```

Comment:
I am not sure if I agree with the reasoning of this rename. I’d probably have gone for `full_lhs_value_xml` instead.